src/compiler/transformers/ts.ts

```diff
@@ -2453,14 +2464,28 @@ namespace ts {
          * @param node The await expression node.
          */
         function visitAwaitExpression(node: AwaitExpression): Expression {

```

Comment:
async/await should be split into its own transform, `es2017.ts`, then we do not need to run it in this case, just like ES7 transform.